USWDS - update v2 readme for npm (#4846)
uswds/uswds: README.md

[![CircleCI Build Status](https://img.shields.io/circleci/build/gh/uswds/uswds/develop?style=for-the-badge&logo=circleci)](https://circleci.com/gh/uswds/uswds/tree/develop) ![Snyk vulnerabilities](https://img.shields.io/snyk/vulnerabilities/npm/uswds?style=for-the-badge) [![npm Version](https://img.shields.io/npm/v/uswds?style=for-the-badge)](https://www.npmjs.com/package/uswds) [![npm Downloads](https://img.shields.io/npm/dt/uswds?style=for-the-badge)](https://www.npmjs.com/package/uswds) [![GitHub issues](https://img.shields.io/github/issues/uswds/uswds?style=for-the-badge&logo=github)](https://github.com/uswds/uswds/issues) [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4?style=for-the-badge)](https://github.com/prettier/prettier)

## :mega: **USWDS 3.0 is now live** :mega:

> Starting with version 3.0, the latest updates to USWDS are now published to the [@uswds/uswds](https://www.npmjs.com/package/@uswds/uswds) package.

Reviewer: Rewritten with active voice and a little more detail

```suggestion
Starting with USWDS 3.0, we're publishing USWDS as a scoped package at [@uswds/uswds].(https://www.npmjs.com/package/@uswds/uswds).
```

> This unscoped `uswds` package will remain active for legacy use.

Reviewer: ```suggestion
This unscoped `uswds` package will remain active for any updates to USWDS 2.x.
```
